tests/e2e/util.go

```diff
@@ -5106,3 +5106,112 @@ func waitForVolumeSnapshotContentToBeDeleted(client snapclient.Clientset, ctx co
+					" | tail -1 | awk '{print $10}' | tr -d '\n'"
+			}
+			framework.Logf("Invoking command '%v' on host %v", grepCmdForFindingCurrentLeader,
+				k8sMasterIP)
+			result, err := sshExec(sshClientConfig, k8sMasterIP,
+				grepCmdForFindingCurrentLeader)
+			if err != nil || result.Code != 0 {
+				fssh.LogResult(result)
+				return "", "", fmt.Errorf("couldn't execute command: %s on host: %v , error: %s",
+					grepCmdForFindingCurrentLeader, k8sMasterIP, err)
+			}
+			if containerName == syncerContainerName {
+				if result.Stdout != "" {
+					csiControllerPod = csiPod.Name
+				}
+			} else {
+				// Fetching the name of pod where container leader is running
+				csiControllerPod = result.Stdout
+			}
+			// If CSI pod  name is found where container leader is running, break from the loop
+			if csiControllerPod != "" {
+				break
+			}
+		}
+	}
+	framework.Logf("CSI pod %s where leader is running", csiControllerPod)
+	// Fetching master node name where container leader is running
+	podData, err := client.CoreV1().Pods(csiSystemNamespace).Get(ctx, csiControllerPod, metav1.GetOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	masterNodeName := podData.Spec.NodeName
+	framework.Logf("Master node name %s where leader is running", masterNodeName)
+	// Fetching IP address of master node where container leader is running
+	k8sNodes, err := client.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	for _, node := range k8sNodes.Items {
+		if node.Name == masterNodeName {
+			addrs := node.Status.Addresses
+			for _, addr := range addrs {
+				if addr.Type == v1.NodeInternalIP && (net.ParseIP(addr.Address)).To4() != nil {
+					k8sMasterNodeIP = addr.Address
+					break
+				}
+			}
+		}
+	}
+	framework.Logf("Master node ip %s where leader is running", k8sMasterIP)
+	return csiControllerPod, k8sMasterNodeIP, nil
+}
+
+// execDockerPauseNKillOnContainer pauses and then kills the particular CSI container on the master node
+func execDockerPauseNKillOnContainer(sshClientConfig *ssh.ClientConfig, k8sMasterNodeIP string,
+	containerName string) error {
+	grepCmdForGettingDockerContainerId := "docker ps | grep " + containerName + " | " +
+		"awk '{print $1}' |  tr -d '\n'"
+	framework.Logf("Invoking command '%v' on host %v", grepCmdForGettingDockerContainerId,
+		k8sMasterNodeIP)
+	dockerContainerInfo, err := sshExec(sshClientConfig, k8sMasterNodeIP,
+		grepCmdForGettingDockerContainerId)
+	fssh.LogResult(dockerContainerInfo)
+	if err != nil || dockerContainerInfo.Code != 0 {
+		return fmt.Errorf("couldn't execute command: %s on host: %v , error: %s",
+			grepCmdForGettingDockerContainerId, k8sMasterNodeIP, err)
+	}
+	grepCmdForPausingDockerContainer := "docker pause " + dockerContainerInfo.Stdout
+	framework.Logf("Invoking command '%v' on host %v", grepCmdForPausingDockerContainer,
+		k8sMasterNodeIP)
+	DockerContainerPauseInfo, err := sshExec(sshClientConfig, k8sMasterNodeIP,
```

Comment:
you can just use `cmdResult` here
'Info' in the name will indicate something else